src/BuiltInTools/BrowserRefresh/WebSocketScriptInjection.js

```diff
@@ -100,6 +100,10 @@ setTimeout(function () {
       .forEach(e => updateCssElement(e));
   }
 
+  function applyBlazorDeltas(deltas) {
+    deltas.forEach(d => window.Blazor._internal.applyHotReload(d.moduleId, d.metadataDelta, d.ilDelta));
```

Comment:
This is a list of items, rather than a single one. I'd like to keep the `applyHotReload` API be as close to `AssemblyExtensions.ApplyDelta` so choosing to do iteration here. In most ordinary case this is a single item.